src/components/ArticleSquare.astro

```diff
@@ -0,0 +1,23 @@
+---
+const { entry } = Astro.props;
+const _date_obj = new Date(entry.data.date);
```

Comment:
Maybe we should find another name for this... my fault, but `_date_obj` is not super descriptive...

Then again it is completely private, internal and temporary.